modules/ROOT/pages/rest-sdk/configure-operations.adoc

```diff
@@ -3,22 +3,35 @@ ifndef::env-site,env-github[]
 include::../_attributes.adoc[]
 endif::[]
 
-== Operations Identifier
-REST SDK heavily relies on the operation identifier for the resources’ classes generation, that operation identifier could be fetched from the OAS model under `operationId` or if it’s from a RAML model it will default to null.
+== Operation Identifiers
 
-The importance of the identifier is that it must not collide with other resources (nor triggers, or any other type of REST SDK component) and that it can be modified in the descriptor by comprehension. REST SDK does provide a default mechanism for generating that resource identifier, in which if there’s a valid operationId (from an OAS context) with a non null value, it will use it, or it execute the fallback of using the HTTP method plus the resource name (which is unique in an API).
+Use an operation identifier in REST SDK to generate classes for resources.
+Fetch an operation identifier from either an OAS model
+under `operationId` or a RAML model. If fetched from a RAML model, the operation
+identifier defaults to null.
 
-But there are scenarios where that default mechanism might need a tweak, and to do so we are externalizing a way to input your own custom Weave script to change how the operations identifiers are generated. The important thing is that there are 3 bindings available for you to use:
+The identifier cannot collide with resources, triggers, or other REST
+SDK components. REST SDK provides a default mechanism to generate an operation
+identifier if there is a valid `operationId` from an OAS context with a non
+null value. It also executes a fallback if there is an HTTP method and
```

Comment:
I feel `It also executes a fallback if there is an HTTP method a...` is missing that it will only be executed if `operationId` is null.